integrations/integration.php

```diff
@@ -102,6 +109,14 @@ public function get_slug(): string {
 		return $this->slug;
```

Comment:
I think you are mistaken by collapsing of code because `get_slug` function returns `string` not `void`, please have a look again and let me know if you still think there is any issue with the code.

Reply:
Oh, right. Sorry for that! 😁